website/control/models.py

```diff
@@ -426,7 +428,7 @@ class ActionLogVesselDetails(models.Model):
 
   # To know the port despite the fact that this may change for the
   # node record itself.
-  node_port = models.IntegerField("Node port", max_length=100, db_index=True)
+  node_port = models.IntegerField("Node port", db_index=True)
```

Comment:
Undid e9ef43b0bb44b879fdd7565d4da93d81789a2e50 in 101289d8d16d26164b4da9fe7d25cb702693cb12.